src/suit/js/SUIT.js

```diff
@@ -30,25 +31,26 @@ var props = {
     ],
 };
 
+function findCorrectTapService() {
+    const url= window.location.href;
+    const idx= url.indexOf('/portal');
```

Comment:
I would be a bit more comfortable with `search('[^/]/portal')` so that we know that if a future deployment is at a URL like https://portal.example.net/ this logic will not improperly remove the hostname.  That would then require using `idx+` two lines down.

Reply:
I am going to revamp this to use `URL`